lazybytez/jojo-discord-bot
# Add Redis as cache provider

## Description
 - Add Redis cache provider
 - Add local Docker stack with Redis and PostgreSQL

## Related issue
Resolves #119 

## How can this be tested?
 - Run the application with in-memory and redis cache modes.
   - Check that the application works in general (log)
   - Run all commands and check all features of the bot (incl. `sync-commands` rate limit) - they should all work properly
   - Check if the webapi endpoints work (multiple reloads of every endpoint!)
   - Check all databasde tables after doing these tests to not contain any broken data
